compiler/x/codegen/OMRInstOpCode.hpp

```diff
@@ -501,6 +527,66 @@ class InstOpCode: public OMR::InstOpCode
    inline uint32_t isFusableCompare()              const { return _properties1[_mnemonic] & IA32OpProp1_FusableCompare; }
    inline bool     isEvexInstruction()             const { return _binaries[_mnemonic].vex_l >> 2 == 1; }
    inline uint32_t getInstructionFeatureHints()    const { return _features[_mnemonic]; }
+   inline uint32_t isVector2Vector()               const { return _properties2[_mnemonic] & IA32OpProp2_Vector2Vector; }
+   inline uint32_t isScalar2Vector()               const { return _properties2[_mnemonic] & IA32OpProp2_Scalar2Vector; }
+   inline uint32_t isSIMDNarrowing()               const { return _properties2[_mnemonic] & IA32OpProp2_SIMDNarrowing; }
+   inline uint32_t canShortenEVEXDisplacement()     const { return isVector2Vector() || isScalar2Vector(); }
+
+   int32_t getSIMDMemOperandSize(OMR::X86::Encoding encoding)
+      {
+      if (canShortenEVEXDisplacement())
+         {
+         int32_t vectorSize = 16;
+         int32_t ratio = 1;
+
+         if (info().isEvex256() || info().isVex256())
+            {
+            vectorSize = 32;
+            }
+
+         if (info().isEvex512())
+            {
+            vectorSize = 64;
+            }
+
+         switch (encoding)
+            {
+            case OMR::X86::VEX_L256:
+            case OMR::X86::EVEX_L256:
+               vectorSize = 32;
+               break;
+            case OMR::X86::EVEX_L512:
+               vectorSize = 64;
+               break;
+            default:
+               break;
+            }
+
+         if (_properties2[_mnemonic] & IA32OpProp2_V2V_RATIO_2x)
```

Comment:
These 4 properties should have inquiry functions for readability and consistency.  e.g.,
```
if (isVector2VectorRatio2x()) ...
```

Reply:
done